README.md

```diff
@@ -333,7 +342,7 @@ The aforementioned guide works well for UI components. However, components like
 
 ## 🖊️ Styling Components
 
-Our customized theme is based on [Design Tokens](https://css-tricks.com/what-are-design-tokens/) using [CSS Variables](https://developer.mozilla.org/en-US/docs/Web/CSS/--*) or a CSS class for each token. Today, we have the following files in the `src/styles` folder:
+Our customized themes is based on [Design Tokens](https://css-tricks.com/what-are-design-tokens/) using [CSS Variables](https://developer.mozilla.org/en-US/docs/Web/CSS/--*) or a CSS class for each token. Today, we have the following files in the `src/styles` folder:
```

Comment:
É um tema ou vários temas?

Reply:
Se forem vários, tem que botar:

```suggestion
Our customized themes are based on [Design Tokens](https://css-tricks.com/what-are-design-tokens/) using [CSS Variables](https://developer.mozilla.org/en-US/docs/Web/CSS/--*) or a CSS class for each token. Today, we have the following files in the `src/styles` folder:
```

Reply:
Atualmente só temos um 👀 @renatamottam

Reply:
I'll accept the suggestion since the plan is to have more than one.